spec/unit/libraries/resource_provider_spec.rb

```diff
@@ -293,7 +293,7 @@ class CatastrophicProvider < OneviewCookbook::ResourceProvider
 
     before do
       expect(res.item).to receive(:retrieve!).and_return(true)
-      allow(res.context.instance_variable_get(:@new_resource)).to receive(:scopes).and_return(['Scope1', 'Scope2'])
+      allow(res.context).to receive(:scopes).and_return(['Scope1', 'Scope2'])
```

Comment:
But this is wrong, the @new_resource is the variable that should receive the scopes